dashboard/lib/policies/user.rb

```diff
@@ -1,4 +1,11 @@
 class Policies::User
+  ALLOWED_EDUCATOR_TYPES = Set[
```

Comment:
This list is not complete, it's put here as a placeholder. Before merging, we should have a complete list of allowed educator types we expect/support.

Reply:
How hard is this to update in the future?

Reply:
Also, how should this be used?

Reply:
not sure if it's helpful, but the pd enrollment form asks the user for their `role` and stores it as json 🤷 

the roles a user sees depend on the course they're enrolling in but the options are:

- School Administrator
- District Administrator
- Parent
- Classroom Teacher
- Media Specialist
- Tech Teacher
- Librarian
- Administrator (don't know the difference between this and a School Administrator 🤔 )
- Counselor
- Other

Reply:
> How hard is this to update in the future?

Easy to update, just add to the list and deploy the PR via DTP. NOTE: removing a role would also require updating the deprecated role in the `serialized_attr` object, which would have it's own set of challenges. It's probably better to be really sure of the roles to add, and add more as necessary rather than starting with a big list and that we end up removing some from.

> Also, how should this be used?

This is used to validate input received from the frontend before we persist it. It could look something like:

```
educator_type = params[:educator_type]
user.educator_type = educator_type if Policies::User::ALLOWED_EDUCATOR_TYPES.includes? educator_type
```